frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -233,6 +233,26 @@ class TagEditorParamInner extends Component<Props> {
     const hasNoDefaultValue = !tag.default;
     const isEmbeddedDisabled = embeddedParameterVisibility === "disabled";
 
+    // We want to remove "default" and "required" so that it
+    // doesn't show up in the default value input update button
+    const defaultParameterInputValue = _.omit(
```

Comment:
nit: I'd try to simplify this construction

Reply:
Agreed. To be fair, it was like that before, I simply extracted it into a variable and did an `omit`, which is the entire reason for all this.